recipe/build.sh

```diff
@@ -1,14 +1,13 @@
 mkdir build
 cd build
-cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+cmake ${CMAKE_ARGS} \
+      -DCMAKE_BUILD_TYPE=RelWithDebInfo \
       -DCMAKE_VERBOSE_MAKEFILE=ON \
-      -DCMAKE_INSTALL_PREFIX=$PREFIX \
-      -DCMAKE_INSTALL_LIBDIR=lib \
```

Comment:
Are these 2 lines no longer required?

On the [conda-forge  knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=compiler%20cxx), it is written :

> Some optional, but useful CMake options:
        ...
        -DCMAKE_INSTALL_PREFIX=$PREFIX Specify the install location.
        -DCMAKE_INSTALL_LIBDIR=lib Libraries will land in $PREFIX/lib, sometimes projects install into lib64 or similar but on conda-forge we keep shared libraries in simply lib.

Reply:
Those 2 options are part of the `CMAKE_ARGS` variable. This is why I removed them.